update.py

```diff
@@ -0,0 +1,8 @@
+from update_firmware import firmware_update
+import logging
+
+firmware_update()
```

Comment:
Instead of calling independent function directly, Create generic class object of Firmware and add respective firmware update methods inside that class.
`Class FirmwareUpgrade(self):
     def update_bios_firmware()
     def update_nic_firmware()
     def update_idrac_firmware()
     def update_all_firmware()
`